tests/js/server/resilience/sharddist/shard-distribution-spec.js

```diff
@@ -45,16 +45,23 @@ let dbServerCount = instanceManager.arangods.filter(arangod => {
 describe('Shard distribution', function () {
 
   before(function() {
+    internal.db._createDatabase(dbname, {replicationVersion: "2"});
```

Comment:
Are we certain to convert this suite for replication2 testing? It will stop testing for replication1. If we want to use this test file for replication2, is it worth adding it to `test-definitions-rlog.txt`?